data/covering_grammar/lib/make_test_file.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
```

Comment:
While I acknowledge you're right that you have a blank line here, let's standardize without it?